celavi/costgraph.py

```diff
@@ -112,7 +112,7 @@ def __init__(
         self.loc_df = pd.read_csv(locations_file)
 
         self.sc_end = sc_end + sc_out_circ + sc_in_circ
-        self.sc_begin = sc_begin
+        self.sc_begin = sc_begin + sc_in_circ
```

Comment:
I thought we had already made that change a while ago when fixing the issue of no panel going to landfills?

Reply:
self.sc_end was the one that got changed before (when nothing was getting landfilled)

The sc_in_circ needed to be in both sc_end and sc_begin as those are the points in the closed loop pathways where components end their previous life and begin their next life.